streams/service/src/main/java/com/hortonworks/streamline/streams/service/UDFCatalogResource.java

```diff
@@ -394,7 +394,7 @@ private Method findMethod(Class<?> clazz, String name) {
     private String uploadJar(InputStream is, String udfName) throws IOException {
         String jarFileName;
         if (is != null) {
-            jarFileName = UUID.randomUUID().toString() + ".jar";
+            jarFileName = String.format("streamline-functions-%s.jar", UUID.randomUUID().toString());
```

Comment:
can't we take the name from uploaded file param and add UUID to it instead of hard-coding here.
We need to do this for custom processor as well